components/gitpod-protocol/src/protocol.ts

```diff
@@ -1011,6 +1011,7 @@ export interface Repository {
 export interface Branch {
     name: string;
     commit: CommitInfo;
+    htmlUrl: string;
```

Comment:
Nit: I'm slightly confused by this attribute name:
- How can a URL be in HTML? 🤔
- Does this mean something like `<a href="...">...</a>`?

I think it's fine to just call it `url` (i.e. the URL of a branch -- pretty self-explanatory)

Reply:
it's called like that in the GH API to distinguish between the URL for cloning and browser navigation. I'd rather like to keep it that way.

Reply:
Hmm, okay, but this is optimizing Gitpod's code readability specifically just for people who know GitHub's API really well (i.e. probably 1 person currently 😛). Those who don't know GitHub's API in depth will probably get confused like me (i.e. the vast majority of current and future Gitpod developers).